__tests__/components/Icon.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Icon } from '../../src/components';
+
+describe('Icon', () => {
+  it('renders a default Icon', () => {
+    const tree = renderer.create(<Icon name={'name'} />);
```

Comment:
`name` is not an icon key, right? does jest create different snapshots for different icons? we could than create tests for our main icons we use across the app.

Reply:
correct brackets in 9d3b4be5b92af2f9b3b9669be38ae70ad29230e1. Jest does not show different snapshots for different icons. What is your idea; to write a test for each icon that is constant in the app? such as left arrow, burger menu, settings .. ?

Reply:
brackets are still `<Icon name={'name'} />` with that commit?

i thought of having different tests for different Icons with different `name` props. but when you say that having different names result in the same snapshot, than this is maybe useless.

on the other hand our icon component can be used in a lot more cases regarding the props. to i could think of a bunch more tests here to care different cases. can you have a look and think of other icon tests to add?

Reply:
"i thought of having different tests for different Icons with different name props. but when you say that having different names result in the same snapshot, than this is maybe useless" 
Here a visual confirmation: 
![Screenshot 2021-03-22 at 15 49 32](https://user-images.githubusercontent.com/43641321/112014339-12c9c600-8b2b-11eb-9a63-9ac869cd3a09.png)
![Screenshot 2021-03-22 at 15 49 20](https://user-images.githubusercontent.com/43641321/112014427-29701d00-8b2b-11eb-8e72-8645f2f355d8.png)

A few more icon cases here 61c542c168e0cf5d5715f440ab76cfabfb8ccf7e 
among with the following warning : 
![Screenshot 2021-03-22 at 18 25 10](https://user-images.githubusercontent.com/43641321/112031758-f8e4af00-8b3b-11eb-8c02-63401087f60f.png)

Reply:
isn't it strange, that all icons result in the same snapshot content? with different props this should variate somehow or what is the expectation?

Reply:
for now we will keep it as is.
these tests will likely not catch many changes, and we should look into why the snapshots are not having more information in the future.

 for now i prioritize moving along with the testing in general before getting hung up on this